app/workers/kms_encryption_verification_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class KmsEncryptionVerificationJob
+  include Sidekiq::Worker
+
+  def perform(models = ApplicationRecord.descendants_using_encryption.map(&:name))
+    models.map(&:constantize).each do |model|
+      keys = model.lockbox_attributes.keys
+      model.where('encrypted_kms_key IS NOT NULL').each do |record|
+        keys.each do |k|
+          box = KmsEncrypted::Box.new
+          decrypted_data_key = box.decrypt(record.encrypted_kms_key,
+                                           context: { model_name: model.to_s, model_id: record.id })
```

Comment:
```suggestion
          decrypted_kms_key = box.decrypt(
            record.encrypted_kms_key, context: { model_name: model.to_s, model_id: record.id }
          )
```

Two suggestions here, rename the variable, and (nit) don't do a super indent on line 13.

Reply:
So this isn't the actual encrypted kms key itself, but rather the data key